templates/storypages/index.html

```diff
@@ -19,20 +19,40 @@
+        
+        <p>Visiting Center City provides a deep dive into the pivotal moments and figures that shaped the United States, offering a tangible connection to the country’s past and its journey towards independence and governance. It serves not just as a physical locale, but as a symbol of the enduring spirit and rich history of the American nation.</p>
       </article>
 
       <article class="slide" id="third-slide">
-        <h2>Third slide title</h2>
-        <p>Pellentesque nunc tellus, euismod sed mauris vitae, volutpat viverra metus. Nunc vel nisl massa. Nunc ullamcorper interdum vehicula. Ut at mi congue, lacinia eros eget, gravida libero. Vivamus et cursus lorem. Nulla elementum neque a nunc sollicitudin, eget maximus turpis egestas. In hac habitasse platea dictumst. Maecenas aliquet nibh tempor leo pretium, vel congue dui malesuada. Sed non felis porttitor, laoreet ante ac, volutpat nibh. Sed tincidunt eleifend felis, et porttitor dui interdum scelerisque. Mauris eget eros pharetra, pretium justo nec, commodo tellus. Duis orci sem, vehicula a leo eget, lacinia venenatis neque. Duis purus enim, volutpat dapibus arcu eget, aliquam malesuada nulla. Duis nisi eros, semper eu bibendum condimentum, vehicula vel felis. Integer a lacus dictum, tempor nulla at, ornare ante.</p>
+        <h2>Independence Bell</h2>
+        <p>Dramatic home of the internationally known symbol of freedom...</p>
+        <img class = "img-eye" src="https://www.visitphilly.com/wp-content/uploads/2019/05/Liberty_Bell_Center_2200vp-1044x781.jpg" width="200"></img>
+        <p>One of the most iconic symbols in U.S. history, the 2,080-pound Liberty Bell stands proudly on Independence Mall. See it up close at the Liberty Bell Center and discover its powerful history — from the night it was rung to announce the Declaration of Independence to its current status as an international emblem of freedom and justice.</p>
+      </article>
+
+      <article class="slide" id="fourth-slide">
+        <h2>Franklin Square</h2>
+        <p>Fun and games in one of William Penn’s original public squares...</p>
+        <img class = "img-eye" src="https://www.visitphilly.com/wp-content/uploads/2022/03/Daytime-Franklin-Square-Fountain-J-Fusco-for-vp-1200x900px-1044x781.jpg" width="200"></img>
+        <p>With a colorful fountain show and activities like mini-golf and carousel rides, Franklin Square offers a carefree respite from the sometimes-serious business of historic sightseeing, but you better believe the historic elements are still there. This delightful patch of green space that lies  north of Independence Hall was one of five public squares laid out by William Penn in his original vision for Philadelphia. From Memorial Day to Labor Day, the park is home to a Once Upon a Nation storytelling bench, featuring short stories about the area from informed historical interpreters.</p>
```

Comment:
I see that a lot of your text content is sourced from visitphilly.com. please cite all your sources or it is considered plagiarism and is not appropriate to display on your portfolio. Example citation:

```suggestion
        <p>With a colorful fountain show and activities like mini-golf and carousel rides, Franklin Square offers a carefree respite from the sometimes-serious business of historic sightseeing, but you better believe the historic elements are still there. This delightful patch of green space that lies  north of Independence Hall was one of five public squares laid out by William Penn in his original vision for Philadelphia. From Memorial Day to Labor Day, the park is home to a Once Upon a Nation storytelling bench, featuring short stories about the area from informed historical interpreters.
        -Source: https://www.visitphilly.com/articles/philadelphia/must-see-historic-attractions-in-historic-philadelphia/
        </p>
```